IdentityCore/src/cache/MSIDKeychainTokenCache.m

```diff
@@ -504,7 +504,7 @@ - (BOOL)removeItemsWithKey:(MSIDCacheKey *)key
     NSData *generic = key.generic;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->